### DISCUSSION TITLE
GoTrue: Allow to send confirmation emails when creating users through admin API

### DISCUSSION THREAD

**Discussion by mtte** [supabase/supabase#15506] [Feature Requests]
## Use case / problem

In our self-hosted Supabase setup, we set `DISABLE_SIGNUP` to `false` for GoTrue, as we are creating the users with our own service. This service makes use of the `createUser` function or the `POST /admin/users` endpoint.

In out case, autoconfim is turned off and users have to verify their email. The problem is, that no confirmation emails are sent when creating a user through the admin API.

## Proposal

Add an optional parameter flag to the `/admin/users` endpoint to trigger confirmation emails to be sent. 

```http
POST /admin/users

{
  [...],
  "send_email_confirmation": true,
}
```

## Workaround

1. Current workaround would be to invite the users instead. This also creates users but sends the invite email instead of the normal email confirmation email. 
2. Use the generate link endpoint and send the emails ourselves
3. Trigger a resend using the resend endpoint (current limitation: no redirect_to parameter can be specified)

**Comment by pieveee** [comment]
We need this!
